src/app/application.cpp

```diff
@@ -102,6 +103,8 @@ namespace
 #define SETTINGS_KEY(name) "Application/" name
 #define FILELOGGER_SETTINGS_KEY(name) (SETTINGS_KEY("FileLogger/") name)
 
+    const int DEFAULT_WORKING_SET = 256; // MiB
```

Comment:
Since no other references it, I would move this to `Application::memoryWorkingSetLimit()`.

Reply:
> Since no other references it, I would move this to `Application::memoryWorkingSetLimit()`.

Inlined in `Application::memoryWorkingSetLimit()`.